generate/generate.go

```diff
@@ -502,15 +503,15 @@ func forEachStack(root, workingDir string, fn forEachStackFunc) Report {
 
 	logger.Trace().Msg("List stacks.")
 
-	stackEntries, err := terramate.ListStacks(root)
+	stacks, err := stack.LoadAll(root)
```

Comment:
for each found stack we walk/parse/load the full list of stacks again?

Reply:
no, it will be loaded once and then it is passed on to the per stack handler. The idea is that forEachStack will call the given function for each stack existing on the project, so this function is called once, we load the stacks, and then call the function for each stack passing the metadata etc.

I could transform this into a loop on generate.Do...it was designed this way because the logic was duplicated on the past, but design changes made it useless, forEachStack is used only in a single place (generate.Do), a simple for would be simpler x_x. Will do in another PR.